src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -124,19 +140,51 @@
     <Compile Include="$(CommonPath)\Interop\Windows\NtDll\Interop.RtlIpv6StringToAddressEx.cs" >
       <Link>Common\Interop\Windows\NtDll\Interop.RtlIpv6StringToAddressEx.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\Interop.SchProtocols.cs" >
-      <Link>Common\Interop\Windows\SChannel\Interop.SchProtocols.cs</Link>
+  </ItemGroup>
+  
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="System\Net\IPAddressPal.Unix.cs" />
```

Comment:
@pgavlin Thanks for the explanation: with no other comments, let's keep the current implementation and pattern throughout the NCL code.